packages/issuer-sdk-js/README.md

```diff
@@ -9,13 +9,123 @@ Get [our NPM package](https://www.npmjs.com/package/@idos-network/idos-sdk) and
 pnpm add @idos-network/idos-issuer-sdk-js
 ```
 
+
+## Quickstart
+
+Create an issuer config with your secret key. This config will be used to interact with the idOS.
+```js
+import { createIssuerConfig } from "@idos-network/idos-issuer-sdk-js";
+import { Wallet } from "ethers";
+
+const wallet = new Wallet("YOUR_PRIVATE_KEY");
+
+const issuerConfig = await createIssuerConfig({
+  nodeUrl: "https://node.url",
+  secretKey: "YOUR_SECRET_KEY",
+  signer: wallet // Or use a KeyPair from near-api-js
+});
+```
+### Creating a human profile.
+
+In order to create a human profile in idOS, you need to have a wallet associated with the human and a public key for the human.
+Passing ids are optional and will be auto-generated if not provided.
+
+```js
+import { createHuman } from "@idos-network/idos-issuer-sdk-js";
+import issuerConfig from "./issuer-config.js";
+
+const human = {
+  id: crypto.randomUUID(),
+  current_public_key: "DERIVED_PUBLIC_KEY",
```

Comment:
I don't think this is self-evident, and we don't instruct the reader on how to discover it.

Also, which public key (current signer? encryption?)? Derived from what?

Reply:
The snippet on the jsdoc makes this a little bit better, but the reader still won't know how to get this value.

Can you get a piece of demo code for that?